labcodes/labstorybook
# DSYS-182: Button playground

**Link to task**
https://labcodes.atlassian.net/browse/DSYS-182

**Description**
This PR implements the `ButtonPlayground` component, used in `Button.stories.mdx`. Also, few small details are refactored across the codebase towards uniformization. For example, `e` as an argument of React methods are replaced by `event`. Occurences of `<>` are replaced by `<React.Fragment>` etc.

**How to test**
Go to `Button` page in the _Stories_. Check if there is a Playground section. Try different input combination.

Notes:
- `light` buttons can't be seen with a white background;
- `Checkbox` has been used instead of `Toggle` due to an easier documentation.
